scripts/uninstall_istio_system.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+if kubectl get --ignore-not-found=true configmap istio-operator-cr-overlay -n istio-system 
+then 
+    istioctl manifest generate -f /app/istio-base.yaml -f /app/overlay-config.yaml | kubectl delete -f -
```

Comment:
I was thinking you'd need to pull these files from configMaps since uninstall doesn't get values, but I suppose since they already exist as configMaps they'd get loaded anyway. You could just check file locally then instead of getting configmap on line 2.